code/drasil-printers/lib/Language/Drasil/Markdown/CreateMd.hs

```diff
@@ -100,7 +100,7 @@ license auth = text "Copyright (c) 2021," <+> auth <>
 
 -- Creates the prefix to be attached to the general Drasil logo path
 buildPath :: String -> String
-buildPath file = [x | x <- pathPrefix, x /= ' ']
+buildPath file = filter (/=' ') pathPrefix
   where
     pathPrefix = unwords $ replicate (numSlash file) "../"
```

Comment:
Now you could even pull `pathPrefix` out of the `where` statement, since the definition of `buildPath` is a lot simpler:

```suggestion
buildPath file = filter (/=' ') $ unwords $ replicate (numSlash file) "../"
  where
```

Reply:
Ok, I will make that change right now!